core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/GroupByAggregator.java

```diff
@@ -61,6 +61,17 @@
     put("VARIANCE", "Variance");
     put("COLLECTLIST", "CollectList");
     put("COLLECTSET", "CollectSet");
+    put("LATESTDATE", "LatestDate");
```

Comment:
why are latestdate and earliestdate needed? These seems like more specifics versions of max and min?

Reply:
this is how it is specified in the requirements doc.

Reply:
Will max and min work for dates as well? If that's the case we don't need to add more functions. But please test the scenario.

Reply:
@bdmogal @albertshau tested with dates and both min and max functions worked. Removed earliest and latest date from PR.

Reply:
Yup that's fine.